src/nplinker/pairedomics/downloader.py

```diff
@@ -203,9 +201,12 @@ def _download_mibig_json(self, version):
         os.makedirs(output_path)
 
         download_and_extract_mibig_metadata(self.project_download_cache,
-                                        output_path, version)
+                                            output_path, version)
 
-        open(os.path.join(output_path, 'completed'), 'w').close()
+        with open(os.path.join(output_path, 'completed'),
```

Comment:
For a better readability, this code could be put to a private method `_create_completed_file`